### DISCUSSION TITLE
Expanding for different country as well.

### DISCUSSION THREAD

**Discussion by rajathv** [OSSPhilippines/philippine-map-app#43] [Q&A]
How can I create the same app for my country like India.

**Comment by warengonzaga** [comment]
You can use the base app actually, then replace it with the SVG to represent your country. Adding @denzdelvillar here for any ideas. cc: @jofftiquez

**Reply by rajathv** [reply]
Hey, thanks for the reply please let me know where you are loading the SVG map.

**Reply by jovi-isorena** [reply]
how did you create the svg's and placed them on the map. Is there a tool that could make this easy?

**Reply by jofftiquez** [reply]
If you guys check this constant variable from `src/utils/constants.js` you'll see how each province are mapped. 

https://github.com/OSSPhilippines/philippines-travel-level-map/blob/a44891f836a00ae2aaee4b8a1a0603dda169cdaf/src/utils/constants.js#L20C1-L522

**Reply by SaiKiru** [reply]
Hi. I developed a [Figma plugin](https://github.com/SaiKiru/philippine-travel-level-map-maker) to make it easier to create and manage custom maps for this.

We can integrate more maps if we so wish but we'll be needing to make changes to how the project is structured as mentioned in #15.

**Reply by jofftiquez** [reply]
> Hi. I developed a [Figma plugin](https://github.com/SaiKiru/philippine-travel-level-map-maker) to make it easier to create and manage custom maps for this.
> 
> We can integrate more maps if we so wish but we'll be needing to make changes to how the project is structured as mentioned in #15.

This is really cool @SaiKiru
